src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -41,8 +41,8 @@ npm add @aws-amplify/ui-react-storage aws-amplify
 import { StorageImage } from '@aws-amplify/ui-react-storage';
 
 export const DefaultStorageImageExample = () => {
-  return <StorageImage alt="cat" path="guest/cat.jpg" />;
-// or <StorageImage alt="cat" path={() => "guest/cat.jpg"} />
+  return <StorageImage alt="cat" path="path-to-img" />;
+// or <StorageImage alt="cat" path={() => "path-to-img"} />
```

Comment:
I don't think we need this, plus isn't this invalid JSX?
```suggestion
```

Reply:
I think it's valid both examples would work, you'd just have to remove the 'or' etc